x/foundation/README.md

```diff
@@ -0,0 +1,1534 @@
+<!--
+order: 0
+title: Foundation Overview
+parent:
+  title: "foundation"
+-->
+
+# `x/foundation`
+
+## Abstract
+
+This module provides the functionalities related to the foundation.
+The foundation can turn off these functionalities irreversibly, through the
+corresponding proposal. Therefore, the users can ensure that no one can bring
+back these foundation-specific functionalities.
+
+## Contents
+
+* [Concepts](#concepts)
+* [State](#state)
+* [Msg Service](#msg-service)
+    * [Msg/UpdateParams](#msgupdateparams)
+    * [Msg/UpdateDecisionPolicy](#msgupdatedecisionpolicy)
+    * [Msg/UpdateMembers](#msgupdatemembers)
+    * [Msg/LeaveFoundation](#msgleavefoundation)
+    * [Msg/SubmitProposal](#msgsubmitproposal)
+    * [Msg/WithdrawProposal](#msgwithdrawproposal)
+    * [Msg/Vote](#msgvote)
+    * [Msg/Exec](#msgexec)
+    * [Msg/Grant](#msggrant)
+    * [Msg/Revoke](#msgrevoke)
+    * [Msg/FundTreasury](#msgfundtreasury)
+    * [Msg/WithdrawFromTreasury](#msgwithdrawfromtreasury)
+* [Events](#events)
+    * [EventUpdateFoundationParams](#eventupdatefoundationparams)
+    * [EventUpdateDecisionPolicy](#eventupdatedecisionpolicy)
+    * [EventUpdateMembers](#eventupdatedmembers)
+    * [EventLeaveFoundation](#eventleavefoundation)
+    * [EventSubmitProposal](#eventsubmitproposal)
+    * [EventWithdrawProposal](#eventwithdrawproposal)
+    * [EventVote](#eventvote)
+    * [EventExec](#eventexec)
+    * [EventGrant](#eventgrant)
+    * [EventRevoke](#eventrevoke)
+    * [EventFundTreasury](#eventfundedtreasury)
+    * [EventWithdrawFromTreasury](#eventwithdrawedfromtreasury)
+* [Client](#client)
+    * [CLI](#cli)
+    * [gRPC](#grpc)
+
+# Concepts
+
+## Foundation Operator
+
+A foundation operator is a module account associated with the foundation and a
+decision policy. It is an "administrator" which has the ability to add, remove
+and update members in the foundation.
+`x/foundation` has several messages which cannot be triggered but by the
+operator. It includes membership management messages, and other messages which
```

Comment:
```
`x/foundation` has several messages which cannot be triggered but by the
 operator.
```
Does It means there is several message only operator can control? I think it is a little difficult. :)

Reply:
Hmm, I don't see any problem. Our tech docs team approved (i.e. didn't modified) this sentence and AFAIK the official docs would use it as is.